hw/ip/lc_ctrl/data/lc_ctrl.hjson

```diff
@@ -612,30 +612,32 @@
-            The individual OTP words are unique and have been engineered so that each word can be incrementally
-            overwritten with another engineered value without causing the ECC bits added by the OTP macro to become
-            inconsistent.
-            '''
-    }
-    { name: "TRANSITION.CTR.SPARSE",
-      desc: '''
-            The life cycle transition counter state is sparsely encoded.
-            Although the encoding is randomly chosen, it satisfies specific Hamming weight and Hamming distance
-            thresholds (see lc_ctrl_state_pkg.sv for the statistics).
-            All counter state encodings (except for the 0 state) have been constructed so that all OTP
-            words belonging to the counter state vector have a non-zero value.
-            The individual OTP words are unique and have been engineered so that each word can be incrementally
-            overwritten with another engineered value without causing the ECC bits added by the OTP macro to become
-            inconsistent.
-            '''
-    }
+    // TODO #26362: Let fusesoc build the cores with all dependencies and then apply the check-countermeasures script
+    // { name: "MANUF.STATE.SPARSE",
+    //   desc: '''
+    //         The manufacturing state vector is sparsely encoded.
+    //         Although the encoding is randomly chosen, it satisfies specific Hamming weight and Hamming distance
+    //         thresholds (see lc_ctrl_state_pkg.sv for the statistics).
+    //         All manufacturing state encodings (except for the RAW state) have been constructed so that all OTP
+    //         words belonging to the manufacturing state vector have a non-zero value.
+    //         The individual OTP words are unique and have been engineered so that each word can be incrementally
+    //         overwritten with another engineered value without causing the ECC bits added by the OTP macro to become
+    //         inconsistent.
+    //         '''
+    // }
+    // TODO #26362: Let fusesoc build the cores with all dependencies and then apply the check-countermeasures script
+    // { name: "TRANSITION.CTR.SPARSE",
+    //   desc: '''
+    //         The life cycle transition counter state is sparsely encoded.
+    //         Although the encoding is randomly chosen, it satisfies specific Hamming weight and Hamming distance
+    //         thresholds (see lc_ctrl_state_pkg.sv for the statistics).
+    //         All counter state encodings (except for the 0 state) have been constructed so that all OTP
+    //         words belonging to the counter state vector have a non-zero value.
+    //         The individual OTP words are unique and have been engineered so that each word can be incrementally
+    //         overwritten with another engineered value without causing the ECC bits added by the OTP macro to become
+    //         inconsistent.
+    //         '''
+    // }
```

Comment:
Instead of removing/disabling the specification/documentation of countermeasures, even if only temporarily, can we waive the failing CI check for the time being?

Reply:
@andreaskurth Wouldn't that shadow all other countermeasure checks? This this is only applied as _one_ test and is appllied on the top.

Reply:
I waived the lc_ctrl IP in topgen

Reply:
This sounds sensible to me, thanks Robert.